src/modules/phoneNumber/__tests__/isValidPhoneNumber.test.ts

```diff
@@ -0,0 +1,48 @@
+import { describe, expect, it } from '@jest/globals';
```

Comment:
Yes these are injected at global level by jest but added this import since I've faced import issue earlier, not in this repo though, added as a failsafe.